go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/PaesslerAG/jsonpath v0.1.1
 	github.com/PuerkitoBio/purell v1.2.0
 	github.com/argoproj/argo-rollouts v1.4.1
+	github.com/cenkalti/backoff v2.2.1+incompatible
```

Comment:
You may have accidentally imported the old version of backoff here (gopls loves to do that). Please make sure you use /v4